docs/src/adr/020adr-prioritization.md

```diff
@@ -0,0 +1,204 @@
+- https://kanbanize.com/lean-management/value-waste/cost-of-delay
+
+## Solution
+
+### Decision
+
+We felt that the CoD approach was too dependent on the need to determine
+short-term financial returns. This is hard to square with our role as an open
+source, R&D center serving Informal Systems and the aims of correct software
+development in the community at large.
+
+We felt that the unstructured voting approach was too subjective. Moreover,
+while it would work to surface our individual preferences, it doesn't help
+establish a common ground for shared understanding about priorities.
+
+We finally decided to adopt the _Action Priority Matrix_. We feel that the
+process is light-weight enough that we can implement it without slowing down our
+development but rigorous enough to root our shared understanding in the
+objective needs and constraints we face in our work.
+
+### Process
+
+We follow an adapted version of [the approach to APM described in the CIO Wiki][apm].
+
+#### Categories
+
+APM leads us to divide work items into four categories, as follows:
+
+| _Impact_ **\\** _Effort_ | **Low Effort** | **High Effort**   |
+|--------------------------|----------------|-------------------|
+| **High Impact**          | "Quick Wins"   | "Major Projects"  |
+| **Low Impact**           | "Fill-ins"     | "Thankless tasks" |
+
+The category within which a work item falls determines the general approach
+taken towards the work:
+
+| Category        | Approach                                           |
+|-----------------|----------------------------------------------------|
+| Quick wins      | Do ASAP                                            |
+| Major projects  | Sustain long term focus                            |
+| Fill-ins        | Do in spare time, but never taking time from above |
+| Thankless tasks | Avoid, unless there's literally nothing else to do |
+
+#### Scoring
+
+We determine which category a work item falls under based on the _impact_ and
+_effort_ scores assigned to the work. We therefore need a shared understanding
+of how to assess the impact and effort of a work item. We register only 3 levels
+on each axis to keep the cognitive load of reckoning scores minimal.
+
+Scores are recorded by affixing labels to the github issue tracking the work item.
+
+##### Effort
+
+Effort is scored best on a rough estimate of the amount of focused time it would
+take to complete a work item:
+
+- Easy **`e0`**: Can be completed within about 1 day
+- Medium **`e1`**: Can be completed within 3 days
+- Difficult **`e2`**: Will take 5 or more days
+
+Ideally, as soon as we recognized that an actively planned ticket will take more
+than 5 days of focused work, we would factor it into smaller tickets, allowing
+us to avoid perpetually prolonged monster tickets. But some people may prefer to
+track "major projects" with a single issue rather than a milestone, and this
+doesn't pose a significant problem.
+
+##### Impact
```

Comment:
Should reasoning be included in the assessments? 

For the impact assessment, including the reasoning can make it clearer to a performer *why* this task is well/poorly aligned with our goals, and thus valuable to complete/not as important as other tasks. 

For the effort assessment, it can help resolve misunderstandings ("This task seems very hard but is scoped as low-effort. Am I misunderstanding the task, or did the person scoping the task underestimate the required effort? Let's see their reason for scoring it as low-effort.") 

However, this adds some overhead when creating work items, so maybe these should only be required for some tasks (maybe e0 i0 could need reasons given?)